services/restund/README.md

```diff
@@ -127,3 +127,17 @@ Example rkt command:
     --user=restund \
     --group=restund
 ```
+
+In case You have set up restund without docker, you just need to make some of these changes:
+
```

Comment:
Please note that the `restund` config file is valid for both running `restund` with `docker` or without